arrow/util.py

```diff
@@ -12,12 +12,31 @@ def is_timestamp(value):
     if isinstance(value, bool):
         return False
     try:
-        datetime.fromtimestamp(value)
+        datetime.datetime.fromtimestamp(value)
         return True
     except TypeError:
         return False
 
 
+# Credit to https://stackoverflow.com/a/1700069
+def iso_to_gregorian(iso_year, iso_week, iso_day):
+    """Converts an iso weekday tuple into a datetime object."""
+
+    if not 1 <= iso_week <= 53:
+        raise ValueError("Iso Calendar week value must be between 1-53.")
```

Comment:
Let's make all the casings of ISO linear in the docstrings and error messages. We should probably capitalize it fully as ISO.